include/RAJA/util/TypedViewBase.hpp

```diff
@@ -101,15 +106,20 @@ namespace internal
   template<typename ... ARGS>
   struct count_num_tensor_args{
     static constexpr camp::idx_t value =
+#if defined(RAJA_ENABLE_EXPTVECTOR)
         RAJA::sum<camp::idx_t>(
             (internal::expt::isTensorIndex<ARGS>() ? 1 : 0) ...);
+#else
+        0;  // There should be 0 Tensor indices if not vectorizing.
+#endif
```

Comment:
Hey @ajkunen, I did not realize that you had refactored `View` so extensively for the Tensor API! I'm trying to separate those tensor parts from the original `View`, and this bit of code is probably the hairiest change. Can you let me know if this is ok? I think everything else in this `TypedViewBase.hpp` file should be ok. Thanks!